### DISCUSSION TITLE
plugin 'adb' not found since last update

### DISCUSSION THREAD

**Discussion by derwaldgeist** [ohmyzsh/ohmyzsh#12577] [Help] [answered]
I just updated oh-my-zsh to the latest version today (I'm always up-to-date), and since then, I am getting this error:

`plugin 'adb' not found`

It worked before, so I assume it's not a configuration error.

**Comment by mcornella** [comment, marked as answer]
We removed the`adb` plugin since Zsh already has official completion built-in. You can remove the error by deleting `adb` from the `plugins` array.

**Comment by derwaldgeist** [comment]
Ok, thanks for the fast response!

**Comment by W-quan** [comment]
But `adb -s` completion is not work after adb plugin removed.

**Reply by mcornella** [reply]
Completing serial devices was included with the first zsh completion in version 4.3.14 https://github.com/zsh-users/zsh/commit/d1a557d008b7fa7881327acbd6decdb50631cc9c#diff-22927d2a058702657e857728852674c37fbc5084c8cb119877b9a81b287d9a7aR399.

Check that `adb devices` is actually returning something, and if it is but the completion is not working, check out the troubleshooting section: https://github.com/ohmyzsh/ohmyzsh/wiki/Troubleshooting#completion-problems.

If nothing helps you can submit a detailed bug report on the zsh-users official mailing list: https://zsh.org/mla.

**Reply by W-quan** [reply]
`adb devices` returns the list of devices correctly.

After do this:
```
compaudit | xargs chmod g-w,o-w
compaudit | xargs chown "$USER"
rm ~/.zcompdump*
exec zsh
```
Still not work.

**Reply by mcornella** [reply]
Please raise a non-OMZ bug report with the trace of the completion function when completing, which you can obtain by:
```
functions -t _adb # set tracing
adb -s [TAB]
```
